src/store/__tests__/useMouStore.test.tsx

```
@@ -1,162 +1,246 @@
+/* ─── Mock immerHistory -------------------------------------------------- */
+/*   commit(draftFn) just performs a deep-clone of the current state,       */
+/*   calls the draftFn, then replaces the zustand state with the clone.     */
+vi.mock("@/utils/immerHistory", () => {
+  // lazy require to avoid circular refs
+  const draftCopy = () => JSON.parse(JSON.stringify(useMouStore.getState()));
+
+  return {
+    history: [],                // not used here, but exported
+    undo: vi.fn(),              // dito
+    redo: vi.fn(),
+    commit: (draftFn: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)